### PR TITLE
[W5.4][T15-A1]CHIA CHERNG HANN

### PR DESCRIPTION
**Pull request by chernghann**

### CODE REVIEW

**Comment by chernghann** [comment]
Contains 5.5b, 5.4, 5.2e, 5.3b

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+
+public class SortCommand extends Command {
```
Comment (K1ang) [inline src/seedu/addressbook/commands/SortCommand.java]: Remember to include header comments for classes and non-trivial methods too!

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -74,34 +64,36 @@ public Command parseCommand(String userInput) {
...
+            case ExitCommand.COMMAND_WORD:
+                return new ExitCommand();
 
-        case HelpCommand.COMMAND_WORD: // Fallthrough
-        default:
-            return new HelpCommand();
+            case HelpCommand.COMMAND_WORD: // Fallthrough
+                default:
+                    return new HelpCommand();
         }
```
Comment (K1ang) [inline src/seedu/addressbook/parser/Parser.java]: Follow this convention for `switch` case. 
https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html

**Comment by K1ang** [comment]
Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/parser/Parser.java (hunk 2 above)
Comment (chernghann) [reply src/seedu/addressbook/parser/Parser.java]: Thank KiangSiang! i will take note!

**Pull request closed without merging**